WalletWasabi.Tests/Helpers/TransactionProcessorExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Blockchain.Keys;
+
+namespace WalletWasabi.Blockchain.TransactionProcessing
```

Comment:
```suggestion
namespace WalletWasabi.Tests.Helpers
```

Reply:
Extensions goes under the namespace of the class it's extending.